extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/photos/GooglePhotosExporter.java

```diff
@@ -153,10 +154,11 @@ public ExportResult<PhotosContainerResource> export(
   }
 
   private ExportResult<PhotosContainerResource> exportPhotosContainer(
-      PhotosContainerResource container, TokensAndUrlAuthData authData)
+      PhotosContainerResource container, TokensAndUrlAuthData authData, UUID jobId)
       throws IOException, InvalidTokenException, PermissionDeniedException {
     ImmutableList.Builder<PhotoAlbum> albumBuilder = ImmutableList.builder();
     ImmutableList.Builder<PhotoModel> photosBuilder = ImmutableList.builder();
+    ImmutableList.Builder<ErrorDetail> errors = ImmutableList.builder();
```

Comment:
move this right before the loop that fills it

Reply:
Done